index.js

```diff
@@ -3,6 +3,8 @@
 var each = require('lodash.foreach');
 var get = require('lodash.get');
 
+var Promise = require('bluebird');
```

Comment:
Is there a reason you went with bluebird instead of native Promises? I know bluebird can often be faster, but I  want to avoid adding production dependencies where possible.